sui-execution/latest/sui-move-natives/src/eth_state_proof.rs

```diff
@@ -124,22 +124,23 @@ pub fn verify_message_proof(
         .map_err(|_| PartialVMError::new(StatusCode::UNKNOWN_STATUS))?;
 
     let contract_address = String::from_utf8(contract_address.clone())
-        .map_err(|_| PartialVMError::new(StatusCode::FAILED_TO_DESERIALIZE_ARGUMENT))?;
+        .unwrap_or(hex::encode(contract_address.clone()));
```

Comment:
Had to do this because the input is a little different between the rust cli and the ts sdk, so if it fails to convert to string, then the hex encoding would do the work